bats/foreman_helper.bash

```diff
@@ -53,6 +53,13 @@ tSkipIfNewerThan45() {
   fi
 }
 
+tContainerPushSupported() {
+  KATELLO_VERSION=$(tKatelloVersion)
+  if ! tIsVersionNewer "${KATELLO_VERSION}" 4.13; then
```

Comment:
That's based on https://github.com/Katello/katello/pull/10952 -- is that a correct assumption?

Reply:
That PR is just the beginning, I need to check but I think 4.14 would be better.

Reply:
@evgeni I confirmed that Katello 4.14 and higher should be used here.

Reply:
done!